tests/eval-one.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/sh
+set -u
+here=$(cd "$(dirname "$0")" && pwd)
+
+if [ -z "$1" ]; then
+  echo "Usage: $0 <profile>"
+  exit 1
+fi
+
+profile=$1
+logfile=$(mktemp)
+
+# shellcheck disable=SC2039
+echo -n "evaluating $profile: "
+
+nix-build \
+  '<nixpkgs/nixos>' \
+  -I "nixos-config=$here/eval-test.nix" \
+  -I "nixos-hardware-profile=$profile" \
+  --dry-run \
+  --no-out-link \
+  --show-trace \
+  -A system > "$logfile" 2>&1
```

Comment:
I don't see the point of having a log file instead of stderr, and I think it hides a lot of potentially useful information. Also, the idea of stderr is exactly to have an error log.

Reply:
No, in the error case the log file contains both stderr and stdin.

Reply:
It doesn't matter, I think `nix-build` only ever writes to `stderr`. What matters is that CI builds may stutter, and with this approach, we would never know why.

Reply:
Also `2>&1` is a bashism.

Reply:
@yegortimoshenko With this approach it's easy to find the failing profile and get to the point instead of going through pages of repeating out links. They might be interesting but not very useful in practice.

Reply:
Right, you only need logs if something goes wrong. Your way of doing things makes certain errors opaque and non-debuggable.